reputation/microeconomic_reputation.tex

```diff
@@ -0,0 +1,367 @@
+The mechanics described in the last conclusion work only when we can estimate the total value of the agreement.
+It's worth noting that such measure is useful also for other purposes, such as estimating the total income from a provider node,
+or just determining if Golem is worth using at all.
+
+\begin{conclusion}
+
+The value of the reputation system depends on how well the reputation-related information improves the "total agreement value" estimation.
+
+\label{agreement value estimation}
+\end{conclusion}
+
+Note that the same reputation system might be valuable for some agreements and useless for other agreements, or even valuable for some agents and useless for others.
+
+\subsection{Optimal strategy dynamics}
+
+Different agents have different utilities and thus different strategies. 
+Also, the market changes because of agents leaving/entering it. We should not expect there to be a single best strategy,
+but rather a constant mix of strategies governed only by a single law: as the time passes, profitable strategies become more common.
+
+As an example, let's assume we have some reputation-related information available to the requestors (e.g. provider benchmarks). 
+If the number of requestors utilizing this information grows, the incentive for providers to have a high value of the benchmark increases.
+If number of such requestors is falling, we should expect more providers not caring about their benchmark values.
+
+\begin{conclusion}
+
+The higher is the adoption of the reputation system (i.e. the more important role it plays in the agents' decisions), the more it influences agents' strategies -
+and thus works better.
+
+\label{adoption level}
+\end{conclusion}
+
+\subsection{Market interventions}
+
+And how to increase the adoption of the reputation system?
+This is a sort of an egg/chicken dillema. Why be honest, when no one cares about honesty? And why care about honesty, when everyone is equally dishonest? \footnote{
+    This sounds extreme from the POV of the "current" market, where most of the providers use the default, honest software, but this is a lower bound
+    towards which our market will be heading without a sufficiently good reputation system.}
+
+The answer is: by market interventions. If we put on the market enough high quality honest agents, who strongly penalize dishonesty 
+(i.e. don't want to trade with dishonest agents), we'll see the balance changing. 
+As the new agents are high quality, everyone will want to trade with them, and this creates two incentives:
+\begin{itemize}
+    \item Our new agents are honest, so there is a reason to use the honesty index as a part of the market strategy (e.g. to find them).
+    \item Our new agents require honesty, so there is a reason to be more honest than the average.
+\end{itemize}
+
+The important thing to note here is that - as both sides change their strategies - the new balance should be preserved even after we remove the new agents from the market.\footnote{
```

Comment:
I don't think this claim is sufficiently substantiated. In fact, the opposite may be the case: if there is plenty of high quality requestors, providers can afford to refuse to deal with lower-quality ones. OTOH when there are few high-quality requestors, insisting on high quality might starve them.

Reply:
I need to think about this.
I've a feeling that this is sort-of-crucial for the reputation system, but I agree this claim requires a stronger support.